stable/postgresql/templates/_helpers.tpl

```diff
@@ -309,7 +309,4 @@ Get the readiness probe command
 {{- else }}
   pg_isready -U {{ include "postgresql.username" . | quote }} -h 127.0.0.1 -p {{ template "postgresql.port" . }}
 {{- end }}
-{{- if contains "bitnami/" .Values.image.repository }}
```

Comment:
I don't think we should delete this validation. 

It was added to prevent the pod to be marked as ready while it is still initializing.